pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -300,17 +298,18 @@ func (c *cephStatusChecker) getRookPodsOnNode(node string) ([]v1.Pod, error) {
 		"csi-cephfsplugin-provisioner",
 		"csi-cephfsplugin",
 		"rook-ceph-operator",
+		"rook-ceph-mon",
+		"rook-ceph-osd",
+		"rook-ceph-crashcollector",
+		"rook-ceph-mgr",
+		"rook-ceph-mds",
 	}
 	podsOnNode := []v1.Pod{}
 	pods, err := c.context.Clientset.CoreV1().Pods(clusterName.Namespace).List(context.TODO(), metav1.ListOptions{})
 	if err != nil {
 		return podsOnNode, errors.Wrapf(err, "failed to get pods on node %q", node)
 	}
 	for _, pod := range pods.Items {
-		if pod.Labels["rook_cluster"] == clusterName.Name {
-			podsOnNode = append(podsOnNode, pod)
-			continue
-		}
```

Comment:
This seems like it changes behavior such that the Rook operator might remove stuck pods from a different Rook operator's cluster. Is this what we want to do? It seems to me that we might want to avoid deleting pods that are from a different `rook_cluster`.

Reply:
but we are fetching pods from the namespace of the current rook-operator. And we have only one `rook_cluster` in one namespace, right?